### PR TITLE
Add contract method to remove a registered address

### PR DESCRIPTION
**Pull request by fvictorio**
Depends on #98.

### CODE REVIEW

File: blockchain/contracts/ProofOfPhysicalAddress.sol [hunk 1]
```diff
@@ -258,6 +258,30 @@ contract ProofOfPhysicalAddress {
...
+        uint256 index;
+        (found, index, ) = userAddressByAddress(msg.sender, country, state, city, location, zip);
+        require(found);
+
+        bytes32 keccakIdentifier = users[msg.sender].physicalAddresses[index].keccakIdentifier;
+        registry.removeClaim(address(this), msg.sender, keccakIdentifier);
+
+        // Remove physical address from list
+        uint256 length = users[msg.sender].physicalAddresses.length;
+        for (uint i = index; i < length - 1; i++){
```
Comment (phahulin) [inline blockchain/contracts/ProofOfPhysicalAddress.sol]: `i` should be `uint256`?

**Comment by phahulin** [comment]
Also, maybe move `blochchain/build` into `.gitignore`?

File: blockchain/contracts/ProofOfPhysicalAddress.sol (hunk 1 above)
Comment (fvictorio) [reply blockchain/contracts/ProofOfPhysicalAddress.sol]: Yes, you are right, thanks.

**Comment by fvictorio** [comment]
> Also, maybe move blochchain/build into .gitignore?

It was already in `.gitignore`, but it was also added for some reason. I removed it.

**Comment by fvictorio** [comment]
This is ready for review now.

**Comment by fvictorio** [comment]
@phahulin @matiasgaratortiz Can we merge this?

**Comment by phahulin** [comment]
@igorbarinov philosophical question: in PoPA contract we have three variables for statistics: `totalUsers`, `totalAddresses`, `totalConfirmed`, incremented respectively when new user is created, when new address is registered and when address is confirmed.

Since it is possible for a user to remove his own claim directly from ERC780 registry, those 3 variables might not represent actual numbers at a given moment.

Now with this PR we also have an ability to remove addresses from PoPA. My question is what should we do about those variables:
1.  we can remove them from contract. Adv: no misleading numbers, Dis: no knowledge on how often the service is used, since mappings cannot be iterated
2. keep them and don't decrement on removal. In this case `totalUsers` will measure how many users ever used PoPA, `totalAddresses` - how many addresses ever in total were registered, etc. Adv: having some statistics. Dis: need to remember that these variables might not represent actual numbers at the present moment, but an overall sum. For example, if a user registers an address and them removes it, we'll still have `totalUser==1`, `totalAddresses == 1`, ...
3. keep them and decrement on removal. Adv: having some statistics, more accurate than in 2. Dis: Still might be not 100% accurate if user manually removes addresses from ERC780 registry

IMO, we should go with 2. @matiasgaratortiz @fvictorio what do you think?

**Comment by fvictorio** [comment]
I agree that 2 seems the better option. I don't think 3 is a good idea, since, as you mentioned, we have no way to have those numbers "synced" with whatever happens in the ERC780 regsitry.

Choosing 2 would also mean that we don't have to make any more changes here, right?

**Comment by phahulin** [comment]
Ok, I think we can merge this now, so that we can continue from here, also, usage of statistical variables is independent of other parts of code.
@igorbarinov if you disagree with choosing (2) option above, please write it here and we can address it in a  separate PR.

**Comment by igorbarinov** [comment]
@phahulin I think 2 is fine

**Pull request merged**